maisim/maisim.Game/Graphics/UserInterface/Overlays/CurrentWorkingBeatmap.cs

```diff
@@ -0,0 +1,64 @@
+using System;
+using maisim.Game.Beatmaps;
+using osu.Framework.Bindables;
+
+namespace maisim.Game.Graphics.UserInterface.Overlays
+{
+    /// <summary>
+    /// A class that provide the bindable for the current <see cref="BeatmapSet"/> and <see cref="DifficultyLevel"/> of the game.
+    /// </summary>
+    public class CurrentWorkingBeatmap
```

Comment:
Would rather have this derive from  `Bindable<Beatmapset>` directly and have any change done to the difficulty level trigger a change of BeatmapSet through `TriggerChanged()`